app/views/certificates.html

```diff
@@ -7,6 +7,9 @@
         <div class="panel-heading">
           <div class="panel-title">
             <h2><i class="glyphicon glyphicon-certificate"></i>&nbsp;&nbsp;Certificates</h2>
+            <p>
+              The OpenHIM keystore is managed here. A new OpenHIM server certificate can be added or generated here and trusted client certificates can also be added or edited. Only self signed client certificates that should be trusted need to be added here. The OpenHIM already trusts all certificates signed by the common CAs. Certificates must be linked to a client on the clients page before they can be used to send in requests.
```

Comment:
It might improve the tone a bit by rewriting the "...here..." statements throughout. E.g. `Manage the OpenHIM keystore.`, `Manage and create channels`.